be/extmap.c

```diff
@@ -1082,14 +1082,14 @@ static int emap_it_open(struct m0_be_emap_cursor *it)
 		ext->ee_val         = rec->er_value;
 		ext->ee_cksum_buf.b_nob  = rec->er_cksum_nob;
 		ext->ee_cksum_buf.b_addr = rec->er_cksum_nob ?
-								 (void *)&rec->er_footer : NULL;
+					   (void *)&rec->er_footer : NULL;
 		it->ec_unit_size = rec->er_unit_size;
  		if (!emap_it_prefix_ok(it))
 			rc = -ESRCH;
 	}
 	it->ec_op.bo_u.u_emap.e_rc = rc;
 
-	return rc;
+	return rc >= 0 ? 0 : M0_ERR(rc);
```

Comment:
No, its format change. Rules for format said to return M0_ERR for error cases so that errors are logged. Please let me know if this is correct else I will remove this.

Reply:
But if `rc > 0` it will return `0`!

Fix it like this: `rc != 0 ? M0_ERR(rc) : rc;` It's more expressive also.

Reply:
okay, thanks @andriytk